[32mINFO    [0m root:test_02_dynamic.py:39 请求成功